src/components/application_manager/src/commands/mobile/subscribe_button_request.cc

```diff
@@ -93,7 +93,8 @@ void SubscribeButtonRequest::Run() {
 bool SubscribeButtonRequest::IsSubscriptionAllowed(
     ApplicationSharedPtr app, mobile_apis::ButtonName::eType btn_id) {
   if (!app->is_media_application() &&
-      ((mobile_apis::ButtonName::SEEKLEFT == btn_id) ||
+      ((mobile_apis::ButtonName::PLAY_PAUSE == btn_id) ||
```

Comment:
@jacobkeeler I beleive this functionality coud be covered with unit tests as well.
The sequence is as following:
1. Create mock non media application
2. CreateSubscribeButton request with PLAY_PAUSE button
3. Call request's Run method
4. Respect should fail